### PR TITLE
Feature/views web UI

### PR DESCRIPTION
**Pull request by adityamukho**
### Scope & Purpose

This PR implements fixes for the Aardvark UI for ArangoSearch Views.

- [x] :hankey: Bugfix
- [x] :pizza: New feature
- [ ] :fire: Performance improvement
- [x] :hammer: Refactoring/simplification

### Checklist

- [ ] Tests
  - [ ] **Regression tests**
  - [ ] C++ **Unit tests**
  - [ ] **integration tests**
  - [ ] **resilience tests**
- [x] :book: CHANGELOG entry made
- [ ] :books: documentation written (release notes, API changes, ...)
- [ ] Backports
  - [ ] Backport for 3.9: *(Please link PR)*
  - [ ] Backport for 3.8: *(Please link PR)*
  - [ ] Backport for 3.7: *(Please link PR)*

#### Related Information

*(Please reference tickets / specification / other PRs etc)*

- [ ] Docs PR: 
- [ ] Enterprise PR:
- [ ] GitHub issue / Jira ticket:
- [x] Design document: https://docs.google.com/document/d/13pm4ckpf-qXpeabZ8kFDamZV3c-EYC9LJMXiUErjOqg/edit#

### CODE REVIEW

**Review by ajurdak — APPROVED** [review]
Nice work! LGTM.

**Review by nikhil-varma — COMMENTED** [review]
Reviewed it. Just a few questions/suggestions. We can consider these for as improvements for future PRs 🙂

File: js/apps/system/_admin/aardvark/APP/react/src/views/views/Components/LinkList.tsx [hunk 1]
```diff
@@ -41,62 +26,74 @@ const LinkList = ({ links, addClick, icon, viewLink }: CollProps) => {
...
+                  </HashLink>
+                </ArangoTD>
+              </tr>
+              </tfoot>
+              : null
+          }
+        </ArangoTable>
+        {
+          isAdminUser && changed
+            ? <SaveButton view={formState} oldName={name} setChanged={setChanged}/>
```
[the remark this answers is not in the document]
Comment (nikhil-varma) [reply js/apps/system/_admin/aardvark/APP/react/src/views/views/Components/LinkList.tsx:94]: Same wherever similar logic is being followed

[the remark this answers is not in the document]
Comment (adityamukho) [reply js/apps/system/_admin/aardvark/APP/react/src/views/views/Components/LinkList.tsx:94]: If JSX is happy, I don't mind switching to the shorter style.

File: js/apps/system/_admin/aardvark/APP/react/src/views/views/Components/ViewLinkLayout.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { ArangoTable, ArangoTD, ArangoTH } from "../../../components/arango/table";
+import { Link } from 'react-router-dom';
+import { ChildProp } from "../../../utils/constants";
+
+type ViewLinkLayoutProps = ChildProp & {
+  fragments?: string[];
+};
+
+const ViewLinkLayout = ({ fragments = [], children }: ViewLinkLayoutProps) =>
```
[the remark this answers is not in the document]
Comment (adityamukho) [reply js/apps/system/_admin/aardvark/APP/react/src/views/views/Components/ViewLinkLayout.tsx:10]: Good point!

File: js/apps/system/_admin/aardvark/APP/react/src/views/views/Components/LinkList.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (nikhil-varma) [reply js/apps/system/_admin/aardvark/APP/react/src/views/views/Components/LinkList.tsx:94]: Yep it'll work

**Review by hkernbach — APPROVED** [review]
Huge PR! LGTM, as far as I can tell (without knowing all the new react based components and mechanisms). Only marked a few sections with old comments, which can be removed.

**Review by KVS85 — CHANGES_REQUESTED** [review]
Need CHANGELOG entry for such a big change.

**Review by KVS85 — APPROVED** [review]
LGTM.

**Pull request merged**